source/install/trouble_mysql.rst

```diff
@@ -0,0 +1,282 @@
+MySQL Installation Troubleshooting
+==================================
+
+ Before you can run the Mattermost server, you must first install and
```

Comment:
Indentation for the opening sentence is not needed here or on the below paragraphs.